app/src/main/java/emu/skyline/MainActivity.kt

```diff
@@ -245,14 +250,14 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
-    private fun sortGameList(gameList : List<AppEntry>) : MutableList<AppEntry> {
+    private fun sortGameList(gameList : List<AppEntry>) : List<AppEntry> {
         val sortedApps : MutableList<AppEntry> = mutableListOf<AppEntry>()
         gameList.forEach { entry -> sortedApps.add(entry) }
         when (preferenceSettings.sortAppsBy) {
-            1 -> sortedApps.sortByDescending { it.name }
-            else -> sortedApps.sortBy { it.name }
+            SortingOrder.AlphabeticalAsc.ordinal -> sortedApps.sortBy { it.name }
+            SortingOrder.AlphabeticalDesc.ordinal -> sortedApps.sortByDescending { it.name }
         }
-        return sortedApps
+        return sortedApps.toList()
```

Comment:
Since `MutableList` inherits from `List`, you don't need to call `toList()` here, you can just return directly.